pkg/convert/testdata/programs/name_overlap/pcl/main.pp

```diff
@@ -0,0 +1,13 @@
+resource "forResourcePcl" "simple:index:resource" {
+  __logicalName = "for_resource_pcl"
```

Comment:
Does the logical name need to be renamed? Couldn't it remain the same?

Reply:
I think it might be better just to keep it consistent (and they are kind of generated from the same field so it'd take more work to make them diverge) which seems like work for confusion.

Reply:
@brandonpollack23 it might make sense to keep the logical name as `for` if people are converting their code, then importing the state from tf state then the references will match